src/components/ChartSetting.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import Menu from './Menu.jsx';
+import { connect } from '../store/Connect';
+import { 
+    EditIcon,
+    PositionLeftIcon,
+    PositionBottomIcon,
+    ThemeDarkIcon,
+    ThemeLightIcon,
+    ChevronRightIcon,
+    BackIcon,
+    FlagIcons
+} from './Icons.jsx';
+import '../../sass/_ciq-chart-setting.scss';
+
+
+const ChartSetting = ({
+    Menu,
+    menuOpen,
+    hasActiveView,
+    clearView,
+    onViewLanguage,
+    isViewLanguage,
+    setLanguage
+}) => {
+    const renderMain = () => {
+        return <div>
+            <div className='title'> {t.translate('Settings')} </div>
+            <div className='body'>
+                <div className="ciq-list ciq-list-setting">
+                    <div className="ciq-list-item">
```

Comment:
`ciq-list-item` will make the cursor to pointer for the whole row; we just want the icons to look clickable.